packages/teams-js/src/public/stageView.ts

```diff
@@ -115,4 +115,39 @@ export namespace stageView {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.stageView ? true : false;
   }
+
+  export namespace stageClose {
+    /**
+     * Closes the current stage view.
+     * @returns Promise that resolves or rejects with an error once the stage view is closed.
+     */
+    export function closeCurrentWindow(): Promise<void> {
+      return new Promise((resolve) => {
+        ensureInitialized(runtime, FrameContexts.stage);
+
+        if (!isSupported()) {
+          throw errorNotSupportedOnPlatform;
+        }
+
+        resolve(
+          sendAndHandleSdkError(
+            getApiVersionTag(stageViewTelemetryVersionNumber, ApiName.StageView_Close),
+            'stageView.closeCurrentWindow',
+          ),
+        );
+      });
+    }
+
+    /**
+     * Checks if stageView capability is supported by the host
+     * @beta
+     * @returns boolean to represent whether the stageView capability is supported
+     *
+     * @throws Error if {@linkcode app.initialize} has not successfully completed
+     *
+     */
+    export function isSupported(): boolean {
+      return ensureInitialized(runtime) && runtime.supports.stageView ? true : false;
```

Comment:
@TrevorJoelHarris for the subcapability should we add a different runtime.support?

Reply:
Yes, you will need a new one for this because hosts can choose to implement this new subcapability independently of whether or not stageView is supported. (which is what makes this not a breaking change for Teams). There are other examples of this already, like `dialog.url` in the runtime object.